packages/components/anchor/src/Anchor.vue

```diff
@@ -0,0 +1,204 @@
+<template>
+  <component :is="wrapperComponent" :target="target" :offset="offsetTop">
+    <div class="ix-anchor-wrapper" :style="wrapperStyle">
+      <div class="ix-anchor">
+        <div class="ix-anchor-ink">
+          <span v-show="showInkInFixed" :class="inkCls" :style="{ top: `${inkTop}px` }" />
+        </div>
+        <slot></slot>
+      </div>
+    </div>
+  </component>
+</template>
+<script lang="ts">
+import {
+  computed,
+  defineComponent,
+  onMounted,
+  onUnmounted,
+  watch,
+  ref,
+  nextTick,
+  provide,
+  Ref,
+  SetupContext,
+  unref,
+} from 'vue'
+import { AnchorProps, Section } from './types'
+import { PropTypes, on, off, isNil } from '@idux/cdk/utils'
+import { getOffsetTop, scrollTo, getCurrentScrollTop } from './util'
+import { getTarget } from '@idux/components/utils'
+
+export default defineComponent({
+  name: 'IxAnchor',
```

Comment:
```suggestion
  name: 'IxAnchor',
  components: { IxAffix },
```
组件内部依赖的其他组件，都需要显式的声明出来，否则用户单组件引入的时候就会报错。
另外你现在的单测是不正确的。